ckan/pastertemplates/template/setup.py_tmpl

```diff
@@ -54,6 +54,7 @@ setup(
     # You can just specify the packages manually here if your project is
     # simple. Or you can use find_packages().
     packages=find_packages(exclude=['contrib', 'docs', 'tests*']),
+    namespace_packages=['ckanext', 'ckanext.{{ project_shortname }}'],
```

Comment:
@wardi: OK, that's fine with me. I will update my PR so that only `ckanext` is declared as a namespace package.